std-bits/table/src/main/java/org/enso/table/aggregations/CountEmpty.java

```diff
@@ -0,0 +1,36 @@
+package org.enso.table.aggregations;
+
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.table.Column;
+import org.enso.table.data.table.problems.InvalidAggregation;
+
+import java.util.List;
+
+/***
+ * Aggregate Column counting the number of (not-)null entries in a group.
+ */
+public class CountEmpty extends AggregateColumn {
```

Comment:
I'd add a short note specifying the behavior depending on `isEmpty` value. Without it, the current name is a bit confusing, at least I was confused when reading it.